package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "build": "webpack",
+    "devserv": "webpack serve",
+    "deploy": "gh-pages -d dist"
```

Comment:
- [ ] **[Optional]:**   `"devserv": "webpack serve --open",` to run the project on the browser automatically.